proof/infoflow/RISCV64/Example_Valid_State.thy

```diff
@@ -1756,15 +1727,15 @@ lemma equal_kernel_mappings_s0[simp]:
                    dest!: kh0_SomeD split: if_splits option.splits)
   apply (clarsimp simp: pts_of_s0)
   apply (clarsimp simp: s0_internal_def riscv_global_pt_def arch_state0_def kh0_obj_def
-                        kernel_mapping_slots_def s0_ptr_defs misc)+
+                        kernel_mapping_slots_def s0_ptr_defs misc elf_index_value)+
   done
 
 lemma valid_asid_map_s0[simp]:
   "valid_asid_map s0_internal"
   by (clarsimp simp: valid_asid_map_def s0_internal_def arch_state0_def)
 
 lemma valid_global_pd_mappings_s0_helper:
-  "\<lbrakk> pptr_base \<le> vref; vref < pptr_base + 0x40000000 \<rbrakk>
+  "\<lbrakk> pptr_base \<le> vref; vref < pptr_base + (1<<kernel_window_bits) \<rbrakk>
```

Comment:
spaces around `<<`?

Reply:
Sure, I'll change any of these that have crept in. I don't think we're completely consistent in putting the spaces in but it's definitely the dominant style.